docs/windows.md

```diff
@@ -0,0 +1,60 @@
+
+### Setup Instructions
+
+**Dependency:** `containerd` v1.7.12+
+
+> **NOTE:** all these requires running as admin (elevated) on a PowerShell terminal.
+
+1. Setup `containerd` by following [the setup instructions here](https://github.com/containerd/containerd/blob/main/docs/getting-started.md#installing-containerd-on-windows). (_Currently, we only support the `containerd` worker_.)
+1. Start the `containerd` service, if not yet started.
+1. Setup `buildkit` binaries:
+    ```powershell
+    $version = "v0.13.0-rc2"
+    $arch = "amd64" # arm64 binary available too
+    curl.exe -LO https://github.com/moby/buildkit/releases/download/$version/buildkit-$version.windows-$arch.tar.gz
+    # there could be another `.\bin` directory from containerd instructions
+    # you can move those
+    mv bin bin2
+    tar.exe xvf .\buildkit-$version.windows-$arch.tar.gz
+    ## x bin/
+    ## x bin/buildctl.exe
+    ## x bin/buildkitd.exe
+    # after the binaries are extracted in the bin directory
+    # move them to an appropriate path in your $Env:PATH directories or:
+    Copy-Item -Path ".\bin" -Destination "$Env:ProgramFiles\buildkit" -Recurse -Force
+    # add `buildkitd.exe` and `buildctl.exe` binaries in the $Env:PATH
+    $Path = [Environment]::GetEnvironmentVariable("PATH", "Machine") + `
+        [IO.Path]::PathSeparator + "$Env:ProgramFiles\buildkit"
+    [Environment]::SetEnvironmentVariable( "Path", $Path, "Machine")
+    $Env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + `
+        [System.Environment]::GetEnvironmentVariable("Path","User")
+    ```
+1. Start `buildkitd.exe`, you should see something similar to:
+
+    ```
+    PS C:\> buildkitd.exe
+    time="2024-02-26T10:42:16+03:00" level=warning msg="using null network as the default"
+    time="2024-02-26T10:42:16+03:00" level=info msg="found worker \"zcy8j5dyjn3gztjv6gv9kn037\", labels=map[org.mobyproject.buildkit.worker.containerd.namespace:buildkit org.mobyproject.buildkit.worker.containerd.uuid:c30661c1-5115-45de-9277-a6386185a283 org.mobyproject.buildkit.worker.executor:containerd org.mobyproject.buildkit.worker.hostname:[deducted] org.mobyproject.buildkit.worker.network: org.mobyproject.buildkit.worker.selinux.enabled:false org.mobyproject.buildkit.worker.snapshotter:windows], platforms=[windows/amd64]"
+    time="2024-02-26T10:42:16+03:00" level=info msg="found 1 workers, default=\"zcy8j5dyjn3gztjv6gv9kn037\""
+    time="2024-02-26T10:42:16+03:00" level=warning msg="currently, only the default worker can be used."
+    time="2024-02-26T10:42:16+03:00" level=info msg="running server on //./pipe/buildkitd"
+    ```
+1. In another terminal (still elevated), try out a `buildctl` command to test that the setup is good:
+    ```
+    PS> buildctl debug info
+    BuildKit: github.com/moby/buildkit v0.0.0+unknown
+    ```
+    > **NOTE:** the version is `v0.0.0+unknown` since this is still a _release candidate (RC)_.
```

Comment:
Add a simple test Dockerfile and command for running through an initial build, and maybe pushing it to a registry.

Reply:
Sure, will do a hello world container.